docs/source/processors/panel.rst

```diff
@@ -8,19 +8,23 @@ You can include an panel using the following text tag:
 .. literalinclude:: ../../../verto/tests/assets/panel/doc_example_basic_usage.md
     :language: none
 
+The title and subtitle are the first Level 1 and Level 2 Headings within the panel.
+They are within the panel rather than given as parameters to make in-context translation simpler and more user friendly.
```

Comment:
The 'in-context' part has no context for developers just reading about Verto. Maybe we could change the wording to:

> They are within the panel rather than given as parameters to allow translation systems to easily identify which text should be translated (for example: the in-context translation system for Crowdin).